client/my-sites/design-preview/index.js

```diff
@@ -94,9 +125,16 @@ const DesignPreview = React.createClass( {
 	},
 
 	loadPreview() {
-		if ( this.props.selectedSiteId && this.props.actions.fetchPreviewMarkup ) {
+		if ( ! config.isEnabled( 'preview-endpoint' ) ) {
```

Comment:
We can check for selectedSite here and remove the check from the two subsequent if statements.